env0/resource_aws_credentials.go

```diff
@@ -98,11 +102,14 @@ func resourceAwsCredentialsCreate(ctx context.Context, d *schema.ResourceData, m
 func resourceAwsCredentialsRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	apiClient := meta.(client.ApiClientInterface)
 
-	id := d.Id()
-	_, err := apiClient.CloudCredentials(id)
+	credentials, err := apiClient.CloudCredentials(d.Id())
 	if err != nil {
 		return ResourceGetFailure("aws credentials", d, err)
 	}
+
+	d.Set("name", credentials.Name)
+	d.SetId(credentials.Id)
```

Comment:
Can we use the seralization logic here instead?

Reply:
yeah sure. In general, I use it when there a multiple fields.
Either way is fine by me. I'll fix it.

Reply:
I think always using it makes a lot of sense since we wouldn't need to change anything when we'll add a new field 💪

Reply:
done!